src/decimal128.ts

```diff
@@ -158,6 +158,32 @@ export class Decimal128 extends BSONValue {
    * @param representation - a numeric string representation.
    */
   static fromString(representation: string): Decimal128 {
+    return Decimal128.fromStringInternal(representation, { allowRounding: false });
```

Comment:
since we don't really have a convention this is optional - but usually to mark something internal we'd either use a free function that isn't exported or use an `_` prefixed method instead: `Decimal128._fromString(..)`.  I prefer the former since it visually distinguishes public versus private methods and members but is cleaner than using "internal".